### ISSUE TITLE
Joining an unnamed room results in CTD

### ISSUE THREAD

**Issue by tecknojock** [WreckedAvent/slimCat#424]
As it says on its face, joining an unnamed room results in such.

**Comment by WreckedAvent** [comment]
Are there any unnamed rooms? I was under the impression a room had to have a name.

**Comment by tecknojock** [comment]
yes, if you create a new room on the web client, it will have a UID but no name until they name it.
